srcpkgs/gvfs/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'gvfs'
 pkgname=gvfs
-version=1.54.2
+version=1.56.1
 revision=1
 build_style=meson
 configure_args="-Dsystemduserunitdir=no -Dtmpfilesdir=no -Dlogind=false
- -Dman=true"
+ -Dman=true -Ddeprecated_apis=true"
```

Comment:
Remove this and make the `-devel` a transitional package:
```
gvfs-devel_package() {
	short_desc+=" - development files (transitional dummy package)"
	build_style=meta
}
```

Nothing in the repos actually needs those headers.